src/Completed.js

```diff
@@ -0,0 +1,19 @@
+import storageManager from './Storage.js';
+import Methods from './Methods.js';
+
+export default class Completed {
+  static markCompleted(checkbox, id, listContainer) {
+    const { checked } = checkbox;
+    const toDoList = storageManager.getData();
+    toDoList[id - 1].completed = checked;
+    storageManager.storeData(toDoList);
+    Methods.uIRefreshInstance(listContainer);
+  }
+
+  static clearCompleted(listContainer) {
+    let toDoList = storageManager.getData();
+    toDoList = toDoList.filter((todo) => todo.completed !== true);
+    storageManager.storeData(toDoList);
+    Methods.uIRefreshInstance(listContainer);
+  }
```

Comment:
- [ ] Please use  Best practice ES6 by using the Arrow functions in all your files.

Reply:
Thank you for your review😊